crt_portal/cts_forms/admin.py

```diff
@@ -145,13 +146,39 @@ class ReportAdmin(ReadOnlyModelAdmin):
     actions = [export_reports_as_csv]
 
 
+class ActorActionFilter(admin.SimpleListFilter):
+    # Human-readable title which will be displayed in the
+    # right admin sidebar just above the filter options.
+    title = 'actor'
+
+    # Parameter for the filter that will be used in the URL query.
+    parameter_name = 'actor_object_id'
+
+    def lookups(self, request, model_admin):
+        users = set([u for u in User.objects.all()])
+        return [(user.id, user.username) for user in users]
+
+    def queryset(self, request, queryset):
+        """
+        Returns the filtered queryset based on the value
+        provided in the query string and retrievable via
+        `self.value()`.
+        """
+        # Compare the requested value (either '80s' or '90s')
+        # to decide how to filter the queryset.
+        if not self.value():
```

Comment:
So we're just checking if self.value is truthy here right? 

This should boil down to: if there is no provided value to the ActorActionFilter, then we just return the queryset, if there is a provided value then we filter the queryset by the provided value, correct?

Reply:
Exactly!

Reply:
Otherwise it does the filter.